metricshub-doc/src/site/markdown/usecases/process-command-line.md

```diff
@@ -0,0 +1,63 @@
+keywords: process command lines, windows
+description: How to monitor process command lines with MetricsHub
+
+# Monitoring a process command line on Windows
```

Comment:
What is important here is not the "command line" part, but the process. We could totally monitor the process by specifying its name, and it would be almost the same thing. We could actually explain that in the use case.